### PR TITLE
LPS-177706 fix the status shown in modal

### PR DESCRIPTION
**Pull request by caiosouza15**

### CODE REVIEW

**Comment by caiosouza15** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/Overview/components/SubscriptionsOverview/components/AccountSubscriptionsList/components/AccountSubscriptionModal/AccountSubscriptionModal.js [hunk 1]
```diff
@@ -35,7 +34,7 @@ const AccountSubscriptionModal = ({
 	] = useOrderItems(externalReferenceCode);
 
 	const totalCount = data?.orderItems.totalCount;
-
+
```
Comment (JoMaAlves) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/Overview/components/SubscriptionsOverview/components/AccountSubscriptionsList/components/AccountSubscriptionModal/AccountSubscriptionModal.js:37]: Please clear this Blank space

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/TeamMembers/components/TeamMembersTable/TeamMembersTable.js [hunk 2]
```diff
@@ -165,7 +167,7 @@ const TeamMembersTable = ({
 			<div className="cp-team-members-table-wrapper overflow-auto">
 				{!totalUserAccounts && !(loading || searching) && (
 					<div className="d-flex justify-content-center pt-4">
-						No team members were found.
+						{i18n.translate("no-team-members-were-found")}
```
Comment (JoMaAlves) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/TeamMembers/components/TeamMembersTable/TeamMembersTable.js:170]: Please, take out the double quotes
```suggestion
						{i18n.translate('no-team-members-were-found')}
```

**Review by JoMaAlves — CHANGES_REQUESTED** [review]
🛠️ Needs some changes 🛠️

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/TeamMembers/components/TeamMembersTable/TeamMembersTable.js [hunk 3]
```diff
@@ -28,6 +29,7 @@ import useUserAccountsByAccountExternalReferenceCode from './hooks/useUserAccoun
 import {getColumns} from './utils/getColumns';
 import getFilteredRoleBriefsByName from './utils/getFilteredRoleBriefsByName';
 
+
```
Comment (JoMaAlves) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/TeamMembers/components/TeamMembersTable/TeamMembersTable.js:32]: Delete this line

**Review by JoMaAlves — CHANGES_REQUESTED** [review]
🛠️ Needs some changes 🛠️

**Review by JoMaAlves — APPROVED** [review]
LGTM ​🦖​🌋​ 
Thank you @caiosouza15

**Comment by JoMaAlves** [comment]
ci:test:sf

**Comment by JoMaAlves** [comment]
ci:test:relevant

**Comment by caiosouza15** [comment]
ci:test:sf

**Comment by caiosouza15** [comment]
ci:test:relevant

**Comment by JoMaAlves** [comment]
Boaa

**Comment by JoMaAlves** [comment]
ci:test:relevant

**Comment by jairmedeiros** [comment]
ci:forward

**Pull request closed without merging**